usefulUtilities/sitV2/appResources/appData/sitClientV2.js

```diff
@@ -339,10 +342,11 @@
                 "addAllOtherSittableOverlays",
                 AvatarList.getAvatarsInRange(_this.seatCenterPosition, CAN_SIT_M)
             );
-            
-            // Enable movement again
-            for (var i in DISABLED_DRIVE_KEYS_DURING_SIT) {
-                MyAvatar.enableDriveKey(DISABLED_DRIVE_KEYS_DURING_SIT[i]);
+            if (settingsEntityID === _this.entityID) {
```

Comment:
this concerns me a bit but i know there will be bugs with this implementation, so let's get this out there first!

Reply:
yeah, there is def an issue around changing seats right now and I think that will need to change but it fixes the more urgent problem with seat floating.